### DISCUSSION TITLE
Problem with combineReducers

### DISCUSSION THREAD

**Discussion by LazarPetrovic1** [reduxjs/redux#4054] [Help] [answered]
Hello everyone.
I've run into a problem and I need a hand.
This is more of a theoretical approach rather than practical.
My problem is:
I am making an app in my spare time, which is rather complex - it's supposed to resemble a social network of sorts.
I have 10 reducers in my `combineReducers` function and a crapload of actions fire up when a user either goes to their dashboard or the start/home page - to fetch the user's uploaded images, videos, their profile picture, posts of their friends, posts directed to them, all of their posts together regardless of whether they are images, videos, audio posts or textual and groups they're a part of.
However, some of the reducers affect others, and simply reset them, but they were supposed to act independently of one another.
In the images below, you can clearly see that some textual posts were loaded in one of the dispatched actions, only to be completely reset by another action from a completely different reducer with a different type constant when loading videos.

![Screenshot from 2021-04-02 16-16-13](https://user-images.githubusercontent.com/47143348/113423528-edf90c80-93ce-11eb-844a-ebbf4ffe7fe3.png)
![Screenshot from 2021-04-02 16-16-16](https://user-images.githubusercontent.com/47143348/113423531-ef2a3980-93ce-11eb-890d-5983205289e9.png)

If anyone knows how to sort this out, I would be grateful. I've been looking for a solution online and have so far come up with nothing

**Comment by markerikson** [comment]
@LazarPetrovic1 Your best bet is probably to ask this on Stack Overflow or in Reactiflux - more people are going to be able to see this and help answer.

**Reply by LazarPetrovic1** [reply]
Alright, thanks

**Comment by LazarPetrovic1** [comment, marked as answer]
Just in case someone stumbles onto this in the future...

Yeah, I solved it, and the problem wasn't in the Redux part, but the React part

**Reply by markerikson** [reply]
Just for reference, what was the actual problem?
